congame-doc/introduction.scrbl

```diff
@@ -327,16 +364,31 @@ Combining all these snippets, the code should look like the below example. Go ah
   [thank-you --> thank-you])
 }|}
 
-@;===============================================
+@;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Test-driving the age study}
 
-@subsection{Previewing the second study}
+Once again, click the @onscreen{Upload Study} button in DrRacket, and (if prompted for the study
+name) enter the name of this new study, @racketvalfont{simple-survey}.
 
-To try out this study, same steps as before: click DrRacket's @onscreen{Run} button, then click near
-the @litchar{>} prompt on the lower pane and type @code{(preview simple-survey)} (note the new name
-of the survey used in the @racket[defstudy] expression).
+@margin-note{If you receive errors at this point, double check that the contents of
+@filepath{age-survey.rkt} exactly match the contents of the example file shown above. In particular,
+check that the top line says @code{#lang conscript} and that the @racket[(provide simple-survey)]
+expression is present in the file.}
 
-Then press your @kbd{ENTER} key. You should see your web browser open with a page that looks roughly
-like this:
+If all goes well, you’ll be taken to a page titled @onscreen{Instances of simple-survey}:
+
+@screenshot{intro-empty-instancelist.png}
+
+As before, click on @onscreen{New Instance} and enter a name for the survey instance --- say,
```

Comment:
This shouldn't be necessary, the old instance merely needs to have its data cleared. Add some note that we add a button to simplify this (orange color margin-note is good).

Reply:
Here we’re actually uploading a new study for the first time (`simple-study` vs `tutorial`) so we do actually need to create an instance for the new study.

Would you prefer the tutorial instruct the reader to re-use the filename and study name from the “simplest study” rather than start a new file? That might simplify the instructions a bit and also give the user a taste of how it works when you edit a study.

Reply:
I see, I was reading too quickly. If it is a new study, then a new file might be the better approach since it usually is, so I would leave it as is, as long as you once cover the simpler update cycle when you want to check a change to an already existing study.